.github/CODEOWNERS

```diff
@@ -7,15 +7,6 @@
 # Default file owners
 * @bitwarden/dept-development-mobile
 
-# DevOps for Actions and other workflow changes
-.github/workflows @bitwarden/dept-devops
-
-# DevOps for Version Bumping
-src/App/Platforms/Android/AndroidManifest.xml
-src/iOS.Autofill/Info.plist
-src/iOS.Extension/Info.plist
-src/iOS.ShareExtension/Info.plist
-src/App/Platforms/iOS/Info.plist
```

Comment:
Please add these back in. The comment can be updated to read `# Shared ownership for version bump automation`